### PR TITLE
Inject TimeService

### PR DESCRIPTION
**Pull request by algattik**
## Purpose
* Use constructor injection to pass mockable TimeService around

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/PlcServer.cs [hunk 1]
```diff
@@ -18,6 +18,12 @@ public partial class PlcServer : StandardServer
         public AlarmConditionServerNodeManager AlarmNodeManager = null;
         public SimpleEventsNodeManager SimpleEventsNodeManager = null;
         public ReferenceNodeManager SimulationNodeManager = null;
+        public readonly TimeService TimeService;
```
Comment (peterbryntesson) [inline src/PlcServer.cs:21]: Shouldn't these all to read-only properties?

**Review by peterbryntesson — COMMENTED** [review]
`LGTM

**Review by peterbryntesson — APPROVED** [review]

**Review by HamletDRC — APPROVED** [review]
looks good to me

File: src/PlcServer.cs (hunk 1 above)
Comment (algattik) [reply src/PlcServer.cs:21]: Sure, but didn’t want to introduce spurious changes in code base that complicate upstream merge. Should get fixed there later

**Pull request merged**